cmd/fake-nvidia-k8s-installer/main.go

```
@@ -0,0 +1,175 @@
+		if !bytes.Equal(current, content) {
+			return fmt.Errorf("refusing to replace unowned CDI spec %s", path)
+		}
+	} else if !errors.Is(err, os.ErrNotExist) {
+		return fmt.Errorf("inspect CDI spec: %w", err)
+	}
+	tmp, err := os.CreateTemp(filepath.Dir(path), ".fake-nvidia-cdi-")
+	if err != nil {
+		return err
+	}
+	tmpName := tmp.Name()
+	defer os.Remove(tmpName)
```

golangci-lint (2.12.2)

[error] Error return value of `os.Remove` is not checked

(errcheck)